Portfolio/index.html

```diff
@@ -33,12 +39,48 @@ <h2>Skills</h2>
         <li>React</li>
         <li>Node js</li>
     </ol>
+    <h2>Projects</h2>
+    <div class="card">
+    <img href="https://cdn.startupsavant.com/images/news/microsoft-chatbot.jpg" alt="chat"/>
+    <h3>Real Time Chat Application</h3>
+    <p>React js,Firebase</p>
+    <p>Developed a real time chat application using firebase and react js</p>
+    <p>User can create,update,delete in real time devices</p>
+    <button class="button">Explore</button>
+    </div>
+    <div class="card">

```

Comment:
There should be a '<head>' tag at the beginning of the HTML code to enclose the link tag.